docs/tutorial/work-pools.md

```diff
@@ -0,0 +1,335 @@
+---
+description: Learn how Prefect deployments can be configured for scheduled and remote execution with work pools.
+tags:
+    - work pools
+    - orchestration
+    - flow runs
+    - deployments
+    - schedules
+    - tutorial
+search:
+  boost: 2
+---
+
+# Work Pools
+
+!!! note "Prefect Cloud"
+    This tutorial uses Prefect Cloud to deploy flows to work pools.
+    Managed execution and push work pools are available in [Prefect Cloud](https://www.prefect.io/cloud) only.
+    If you are not using Prefect Cloud, please learn about work pools below and then proceed to the [next tutorial](/tutorial/workers/) that uses worker-based work pools.
+
+## Why work pools?
+
+Work pools are a bridge between the Prefect orchestration layer and infrastructure for flow runs that can be dynamically provisioned.
```

Comment:
This statement feels less true now that we have managed execution. We may want to simply present work pools as places your flow code can run.